test_env/config/deploy.rb

```diff
@@ -1,31 +1,29 @@
-require 'mina'
+require 'mina/rails'
 require 'mina/git'
-require 'mina_sidekiq/tasks'
-require 'mina/git'
-require 'mina/bundler'
 require 'mina/rvm'
+require 'mina_sidekiq/tasks'
 require 'fileutils'
 
-
 FileUtils.mkdir_p "#{Dir.pwd}/deploy"
 
 set :ssh_options, '-o StrictHostKeyChecking=no'
 
 set :domain, 'localhost'
 set :deploy_to, "#{Dir.pwd}/deploy"
-set :repository, 'https://github.com/Mic92/mina-sidekiq-test-rails.git'
+set :repository, 'https://github.com/devvmh-forks/mina-sidekiq-test-rails.git'
```

Comment:
so the only reason travis ci is not working, because the ssh key is not added to travis ci?